crAPI/api-specs/identity/openapi.json

```diff
@@ -563,6 +563,27 @@
                 "security": []
             }
         },
+        "/identity/privacy/user_agent": {
```

Comment:
@ricekot you also need to add the endpoint in `demo-scenario-n-ok.json` and `onboarding-scenarios.json` files in the demo scenarios folder.

Reply:
Done.